### PR TITLE
Allow overriding of .json config option.

### PR DESCRIPTION
**Pull request by pmark**

### CODE REVIEW

**Comment by etburke** [comment]
Resolves #20

**Comment by cmwood** [comment]
Cool. Tests?

File: index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ Datahub.prototype._crud = function (url, method, data) {
   }
 
   if (method === 'GET') {
-    options.json = true;
+    options.json = (options.json == null) ? true : options.json;
```
Comment (cmwood) [inline index.js]: options.json = (!options.json) ? true : options.json;

Will this work for you?

Comment (etburke) [inline index.js]: We thought the same thing at first, but...
!false === true

Comment (cmwood) [inline index.js]: Ah, right.

**Pull request merged**